rendered/zip-0253.html

```diff
@@ -68,9 +68,9 @@ <h1 id="references">References</h1>
 <li id="fn3" role="doc-endnote"><p><a href="protocol/protocol.pdf#networks">Zcash Protocol Specification, Version v2023.4.0 or later. Section 3.12: Mainnet and Testnet</a><a href="#fnref3" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
 <li id="fn4" role="doc-endnote"><p><a href="protocol/protocol.pdf">Zcash Protocol Specification, Version v2023.4.0 or later</a><a href="#fnref4" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
 <li id="fn5" role="doc-endnote"><p><a href="zip-0200">ZIP 200: Network Upgrade Mechanism</a><a href="#fnref5" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
```

Comment:
References are not being deduplicated in the rendered HTML. It's not necessary to fix it in this PR, but I've filed #893.